build/DependencyVersions.props

```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.4.0-preview3-4475</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25708-01</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
```

Comment:
This seems backwards.  @weshaggard or @ericstj may be able to confirm which version we should be using.

Reply:
This is because you reverted it at the end of 2.0.2. This accurately represents the state of 2.0.x. You only put back the better version in 15.5.x